src/config/defaults.ts

```diff
@@ -107,14 +107,14 @@ const canadaCities: string[] = [
     "Home based - Canada, Toronto",
     "Home based - Canada, Mississauga",
     "Home based - Canada, Hamilton",
-    "Home based - Canada, Sudbury",
+    // "Home based - Canada, Sudbury",
```

Comment:
I'd maybe add a comment here so we don't forget that this has been commented out for a reason